src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -465,6 +505,53 @@ private void ParseRuleFromDescription()
             _ruleDictionary.Add(rule.Id, rule);
         }
 
+        private void ParseNodes()
+        {
+            _reader.Read();
+            while (!AtEndOf(_strings.UnifiedNodePool))
+            {
+                if (AtStartOfNonEmpty(_strings.Node))
+                {
+                    ParseNode();
+                }
+                else
+                {
+                    _reader.Read();
+                }
+            }
+        }
+
+        private void ParseNode()
+        {
+            string nodeId = _reader.GetAttribute(_strings.IdAttribute);
+            _reader.Read();
+            while (!AtEndOf(_strings.Node))
+            {
+                if (AtStartOf(_strings.SourceLocation))
+                {
+                    PhysicalLocation physLoc = ParsePhysicalLocationFromSourceInfo();
+
+                    // Step past the empty SourceLocation element.
+                    _reader.Read();
+
+                    // Get the node text from the Action element
+                    string nodeLabel = _reader.ReadElementContentAsString();
+
+                    // Create the region's snippet
+                    physLoc.Region.Snippet = new FileContent
+                    {
+                        Text = nodeLabel
```

Comment:
>nodeLabel [](start = 31, length = 9)

Is this source code? Same comment as previously, it should be, or this should be an annotation.